frontend/src/components/application/editAppFilingName/EditAppFilingName.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import { useTranslations } from "next-intl";
+import { useRef } from "react";
+import { ModalRef, ModalToggleButton } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+import { EditAppFilingNameModal } from "./EditAppFilingNameModal";
+
+interface EditAppFilingNameProps {
+  applicationId: string;
+  applicationName: string;
+  opportunityName: string | null;
+}
+
+export const EditAppFilingName = ({
+  applicationId,
+  applicationName,
+  opportunityName,
+}: EditAppFilingNameProps) => {
+  const t = useTranslations("Application");
```

Comment:
[nit] I noticed you did this elsewhere, but you could drill down farther here
```suggestion
  const t = useTranslations("Application.information.editApplicationFilingNameModal");
```